src/demo-app/tabs/tabs-demo.html

```diff
@@ -288,3 +288,17 @@ <h1>Tabs with autosize textarea</h1>
     </div>
   </mat-tab>
 </mat-tab-group>
+
+<h1>Lazy Loaded Tabs</h1>
+<mat-tab-group>
+  <mat-tab label="First">
```

Comment:
Would be neat to see a third tab that doesn't use lazy-loading so we can see the difference somehow (e.g. the content of Counter component should have the time it was loaded?)